### PR TITLE
Updated Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
:-)

### CODE REVIEW

File: src/translations/installer_nl.ts [hunk 1]
```diff
@@ -182,43 +182,43 @@
...
     </message>
     <message>
         <source>OpenUrl</source>
-        <translation>Opening $0 in default browser...</translation>
+        <translation>Bezig met openen van $0...</translation>
     </message>
     <message>
         <source>MissingVcRuntime</source>
-        <translation>Unable to find Visual C++ Runtime ${ARCH}.</translation>
+        <translation>Visual C++ Runtime ${ARCH} ontbreekt</translation>
```
Comment (Abestanis) [inline src/translations/installer_nl.ts:197]:
```suggestion
        <translation>Visual C++ Runtime ${ARCH} ontbreekt.</translation>
```

File: src/translations/installer_nl.ts [hunk 2]
```diff
@@ -182,43 +182,43 @@
...
     </message>
     <message>
         <source>MissingVcRuntime_StillNotFound</source>
-        <translation>The required component was not found on your computer. Please search online for Visual C++ ${ARCH} downloads for Visual Studio 2015 and higher.</translation>
+        <translation>Het vereiste onderdeel is nog steeds niet aangetroffen op uw computer. Zoek op internet naar Visual C++ ${ARCH}-downloads voor Visual Studio 2015 of nieuwer.</translation>
     </message>
     <message>
         <source>MissingVcRuntime_UnableToRun</source>
-        <translation>${PRODUCT_NAME} will not be able to run.</translation>
+        <translation>${PRODUCT_NAME}kan niet worden gestart.</translation>
```
Comment (Abestanis) [inline src/translations/installer_nl.ts:217]:
```suggestion
        <translation>${PRODUCT_NAME} kan niet worden gestart.</translation>
```

File: src/translations/main_nl.ts [hunk 3]
```diff
@@ -421,47 +421,47 @@ p, li { white-space: pre-wrap; }
     </message>
     <message>
         <source>opaque when new mail is present,</source>
-        <translation type="unfinished"></translation>
+        <translation>ondoorzichtig als er nieuwe e-mails zijn;</translation>
```
Comment (Abestanis) [inline src/translations/main_nl.ts:424]:
```suggestion
        <translation>ondoorzichtig als er nieuwe e-mails zijn,</translation>
```
I'm unsure here, did you want the semicolon or was it accidental?

File: src/translations/main_nl.ts [hunk 4]
```diff
@@ -421,47 +421,47 @@ p, li { white-space: pre-wrap; }
...
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kruis dit aan om de optie &amp;quot;Huidige ongelezen e-mails negeren &amp;quot; toe te voegen aan het rechtermuisknopmenu. Birdtray doet dan alsof er geen ongelezen e-mails meer zijn totdat er nieuwe e-mails binnenkomen.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;Voorbeeld: als er 10 ongelezen e-mails zijn en u heeft de optie &amp;quot;Negeren&amp;quot; geactiveerd, dan toont Birdtray geen indicator zolang het aantal op 10 blijft. Zodra er weer nieuwe e-mails binnenkomen, toont Birdtray het aantal vanaf dan, beginnende bij 1.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;Als het aantal ongelezen e-mails lager wordt dan het aantal &amp;quot;genegeerde&amp;quot;, dan wordt de teller op 0 gezet.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```
Comment (Abestanis) [inline src/translations/main_nl.ts:448]:
```suggestion
        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kruis dit aan om de optie &amp;quot;Huidige ongelezen e-mails negeren &amp;quot; toe te voegen aan het rechtermuisknopmenu. Birdtray doet dan alsof er geen ongelezen e-mails meer zijn totdat er nieuwe e-mails binnenkomen.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;Voorbeeld: Als er 10 ongelezen e-mails zijn en u heeft de optie &amp;quot;Negeren&amp;quot; geactiveerd, dan toont Birdtray geen indicator zolang het aantal op 10 blijft. Zodra er weer nieuwe e-mails binnenkomen, toont Birdtray het aantal vanaf dan, beginnende bij 1.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;Als het aantal ongelezen e-mails lager wordt dan het aantal &amp;quot;genegeerde&amp;quot;, dan wordt de teller op 0 gezet.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```
[...] Voorbeeld: **A**ls er 10 [...]

File: src/translations/main_nl.ts (hunk 3 above)
Comment (Vistaus) [reply src/translations/main_nl.ts:424]: I thought the string below it followed this one, so I read the sentence out loud to determine if a comma or semicolon would sound better and decided to go with the semicolon. So it wasn't accidental.

File: src/translations/main_nl.ts (hunk 4 above)
Comment (Vistaus) [reply src/translations/main_nl.ts:448]: Dutch grammar rules approve both, but a capital might be a bit nicer after all, so I'll commit this.

**Comment by Vistaus** [comment]
> Thank you for updating the translation

No problem :-)

**Pull request merged**